### PR TITLE
Feat/add dagda challenges

### PR DESCRIPTION
**Pull request by houssemmimoun27**
This PR adds the dagda challenges for Builders Program.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -42,10 +43,23 @@ OKP4_GITHUB_PREDICATE_ADDRESS_URL='https://github.com/okp4/okp4d/blob/v4.1.0/x/l
...
+OKP4_GITHUB_ONTOLOGY_CSV_DATASET_URL='https://github.com/okp4/ontology/blob/main/src/core.ttl#L44'
+OKP4_GITHUB_ONTOLOGY_CSV_DATASET_METADATA_URL='https://github.com/okp4/ontology/blob/main/src/metadata-dataset-general.ttl'
+OKP4_GITHUB_SERVICE_REFERENCE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/service/d1b0b4d3-f9a6-4115-bcd8-ad97233a7b08.ttl'
+OKP4_GITHUB_DATASET_REFERENCE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/dataset/1a4c5896-b13e-40f7-a603-5a5b3ed996a1.ttl'
+OKP4_GITHUB_COGNITARIUM_PROLOG_CODE_URL='https://github.com/okp4/contracts/blob/main/contracts/okp4-cognitarium/examples/prolog-query/query.pl'
+OKP4_GITHUB_COGNITARIUM_SERVICE_CATEGORY_URL='https://github.com/okp4/ontology/blob/main/src/vocab-category-service.ttl'
+OKP4_GITHUB_COGNITARIUM_DATASET_MEDIATYPE_URL='https://github.com/okp4/ontology/blob/main/src/vocab-mediatype.ttl'
+OKP4_GITHUB_STORAGE_SERVICE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/service/d1b0b4d3-f9a6-4115-bcd8-ad97233a7b08.ttl#L10'
+OKP4_GITHUB_DATASET_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/dataset/1a4c5896-b13e-40f7-a603-5a5b3ed996a1.ttl#L21C10-L21C10'
```
Comment (ccamel) [inline .env:56]: The links are not correct. Use persistent links based on Git hashes or tags to ensure the stability and prevent broken or outdated references. It's especially important when you're designating a line in a source file that's likely to change the next time you make a change.

File: .env [hunk 2]
```diff
@@ -55,6 +69,15 @@ OKP4_FAUCET_URL='https://faucet.okp4.network/'
 OKP4_SWI_PROLOG_URL='https://www.swi-prolog.org/'
 OKP4_SWI_PROLOG_FUNCTIONS_URL='https://www.swi-prolog.org/pldoc/man?section=functions'
 OKP4_ONTOLOGY_DESCRIPTION_URL='https://blog.okp4.network/okp4-knowledge-sharing-through-ontology-driven-dataverse-a-new-approach-to-knowledge-description-f9b81d4484c8'
+CSV_DATASET_URL='https://data.europa.eu/data/datasets/d043d1f8-f73e-47d1-a471-53156f5e051b?locale=en'
+AWS_S3_URL='https://aws.amazon.com/fr/s3/'
+SCALEWAY_OBJECT_STORAGE_URL='https://www.scaleway.com/fr/object-storage/'
+UUID_IDENTIFIER_URL='https://uuidonline.com/'
```
Comment (ccamel) [inline .env:75]: I'd be inclined to promote an opensource project for online `uuid` generation,like [aarreedd/uuidtools](https://github.com/aarreedd/uuidtools.com) - <https://www.uuidtools.com/>. But it's debatable.

File: src/data/phase/samhain.tsx [hunk 3]
```diff
@@ -842,6 +866,673 @@ export const samhain = ({
           }
         }
       ]
+    },
+    {
+      challengeName: 'Invoke the power of Dagda- From July, 12th to August, 30th',
```
Comment (ccamel) [inline src/data/phase/samhain.tsx:871]:
```suggestion
      challengeName: 'Invoke the power of Dagda - From July, 12th to August, 30th',
```

**Review by ccamel — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (antho31) [reply .env:56]: I updated the links [here](https://github.com/okp4/dev-kanban/issues/1551), but too late. 
Should we provide persistent links for each GitHub link, or only for those with line anchors?

Comment (antho31) [reply .env:56]: Links proposal: 

```
OKP4_GITHUB_ONTOLOGY_S3_SERVICE_URL='https://github.com/okp4/ontology/blob/main/src/core.ttl'
OKP4_GITHUB_S3_SERVICE_METADATA_URL='https://github.com/okp4/ontology/blob/main/src/metadata-service-general.ttl'
OKP4_GITHUB_ONTOLOGY_CSV_DATASET_URL='https://github.com/okp4/ontology/blob/main/src/core.ttl'
OKP4_GITHUB_ONTOLOGY_CSV_DATASET_METADATA_URL='https://github.com/okp4/ontology/blob/main/src/metadata-dataset-general.ttl'
OKP4_GITHUB_SERVICE_REFERENCE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/service/d1b0b4d3-f9a6-4115-bcd8-ad97233a7b08.ttl'
OKP4_GITHUB_DATASET_REFERENCE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/dataset/1a4c5896-b13e-40f7-a603-5a5b3ed996a1.ttl'
OKP4_GITHUB_COGNITARIUM_PROLOG_CODE_URL='https://github.com/okp4/contracts/blob/main/contracts/okp4-cognitarium/examples/prolog-query/query.pl'
OKP4_GITHUB_COGNITARIUM_SERVICE_CATEGORY_URL='https://github.com/okp4/ontology/blob/main/src/vocab-category-service.ttl'
OKP4_GITHUB_COGNITARIUM_DATASET_MEDIATYPE_URL='https://github.com/okp4/ontology/blob/main/src/vocab-mediatype.ttl'
OKP4_GITHUB_STORAGE_SERVICE_URL='https://github.com/okp4/ontology/blob/d3209eb9395a0292927627004e5114b63bf3cab0/example/rhizome/service/d1b0b4d3-f9a6-4115-bcd8-ad97233a7b08.ttl#L10'
OKP4_GITHUB_DATASET_URL='https://github.com/okp4/ontology/blob/d3209eb9395a0292927627004e5114b63bf3cab0/example/rhizome/dataset/1a4c5896-b13e-40f7-a603-5a5b3ed996a1.ttl#L21C20-L21C39'
```

File: .env (hunk 2 above)
Comment (antho31) [reply .env:75]: I provided a random project, you're right. Let use https://github.com/aarreedd/uuidtools.com for `UUID_IDENTIFIER_URL`

**Comment by fredericvilcot** [comment]
@antho31 @ccamel 
@houssemmimoun27 made the changes about the env variables, could you check and tell us if everything looks good to you?

**Comment by antho31** [comment]
I see one thing: the italic style needs to be applied. Once fixed, OK!

**Comment by houssemmimoun27** [comment]
**FYI** @fredericvilcot @antho31  @ccamel 
This PR must be deployed on staging today

**Review by fredericvilcot — APPROVED** [review]
Great 👍

**Review by antho31 — APPROVED** [review]

**Review by ccamel — APPROVED** [review]
LGTM!

**Pull request merged**